free-programming-books.md

```diff
@@ -2408,6 +2408,7 @@ Kerridge (PDF) (email address *requested*, not required)
 #### Flask
 
 * [Explore Flask](https://exploreflask.com) - Robert Picard
+* [Flask Documentation](https://flask.palletsprojects.com/en/1.1.x/) - Pallets (:construction: *in process*)
```

Comment:
It might be better to prefer a link that doesn't mention the language or version? 
Based on how older versions act in the URL, this link might eventually break when or if it moves to 1.2.x.
Meanwhile, linking to the homepage redirects to the correct page anyway.

```suggestion
* [Flask Documentation](https://flask.palletsprojects.com) - Pallets (:construction: *in process*)
```